libs/accounts/src/lib/accounts-table.tsx

```diff
@@ -23,19 +25,42 @@ export interface Datasource extends IDatasource {
   getRows(params: GetRowsParams): void;
 }
 
+export type MarketAsset = Pick<Asset, 'symbol' | 'name' | 'id' | 'decimals'>;
+
 export interface AccountTableProps extends AgGridReactProps {
   rowData?: AccountFields[] | null;
   datasource?: Datasource;
   onClickAsset: (assetId: string) => void;
   onClickWithdraw?: (assetId: string) => void;
   onClickDeposit?: (assetId: string) => void;
   isReadOnly: boolean;
+  marketAsset?: MarketAsset;
```

Comment:
suggestion (non blocking): I think a better name for this prop would just be `pinnedAsset`. It more clearly describes what will happen if you use it and doesn't conceptually tie it to a market